index.html

```diff
@@ -189,5 +184,58 @@ <h5>Skills</h5>
         </li>
       </ul>
     </section>
+
+    <!--forms section-->
+
+    <div class="form-container">
+      <p class="form-intro">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </p>
+      <form action="https://formspree.io/f/mwkzpvyn" method="POST">
+       
+           <input type="text" name="name" class="name" placeholder="Full name" maxlength="30">
+       
+            <input type="email" name="email" class="email" placeholder="Email address">
+            <textarea name="your-text" class="your-text" cols="30" rows="10" maxlength="500" placeholder="Write your message here"></textarea>
+          <div class="bottom-button">
+            <button type="button">Get in Touch</button>
+        </div>
+      </form>
+
+      <div class="line"></div>
+      <div class="socials-bottom">
+      <ul class="socials-bottom-list">
+        <li class="github">
+          <a href="https://github.com">
+           <img src="./assets/svg/github-dark.svg" alt="github">
+          </a>
+        </li>
+        <li class="linkedin">
+            <a href="https://linkedin.com">
+              <img src="./assets/svg/linkedin-dark.svg" alt="linkedin">
+            </a>
+        </li>
+        
+        <li class="angelist">
+            <a href="https://angelist.com">              
+             <img src="./assets/svg/angelist-dark.svg" alt="angelist">
+            </a>
+        </li>
+        <li class="twitter">
+            <a href="http://www.twitter.com/mykved">
+                <img src="./assets/svg/twitter-dark.svg" alt="twitter">
+            </a>
+        </li>
+            
+        <li class="medium">
+            <a href="https://medium.com">
+        <img src="./assets/svg/medium-dark.svg" alt="medium">
+            </a>
+        </li>             
+      </ul>               
+   </div>
```

Comment:
- [OPTIONAL] I suggest using semantic HTML5 tag such as `<footer>` instead of  `<div>` to follow the [best practices for HTML](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). 💡